common/result.c

```diff
@@ -96,8 +96,8 @@ const char* oe_result_str(oe_result_t result)
             return "OE_INVALID_UTC_DATE_TIME";
         case OE_INVALID_QE_IDENTITY_INFO:
             return "OE_INVALID_QE_IDENTITY_INFO";
-        case OE_INVALID_COLLATERAL:
-            return "OE_INVALID_COLLATERAL";
+        case OE_INVALID_SGX_COLLATERAL:
```

Comment:
Question: why can't this value stay TEE-agnostic?   Do really need a separate error code for every supported TEE?  I would prefer keeping it TEE-agnostic if at all possible, e.g., OE_INVALID_ENDORSEMENT

Reply:
Thanks, has changed to OE_INVALID_ENDORSEMENT